docs/source/operations/fabric_verify.md

```diff
@@ -1,19 +1,19 @@
-# How to debug a Fabric deployment
+# How to debug a BAF deployment

```

Comment:
File name needs to be changed as this is not only "fabric_verify" now

Reply:
done